### ISSUE TITLE
502 Bad Gateway on master instance and XTRAS

### ISSUE THREAD

**Issue by leevh** [omega8cc/boa#846]
Hello, I am getting 502 on master aegir instance, and all XTRAS (cgp, chive, and just added sqlbuddy).  Octopus instance and all sites are fine.  "boa info more" gist is below.  I'm not sure at what upgrade this occurred as I don't use the xtras or master instance very often.  Nothing non standard was done intentionally.  I'm not sure what other info/logs to supply, but please let me know what would help and many thanks for any suggestions :)

Running on linode 4096
https://gist.github.com/leevh/e4fac21c0ed7251ecee1

cheers

**Comment by omega8cc** [comment]
You should check errors in the `/var/log/php/*` to see what happens.

**Comment by leevh** [comment]
I found nothing there, but also checked nginx logs and found many errors about www55.fpm.socket not being found.  I ended up having to edit nginx_vhost_common.conf and nginx_compact_include.conf and replace all mention of www55.fpm.socket to www56.fpm.socket.  I recently switched to single install php5.6.  

In case you're curious, the procedure I followed to upgrade to 5.6 was:
- added 5.6 to the multi-install in .barracuda.cnf
- ran up stable system
- switched my octopus control files to 5.6
- once all sites showed 5.6, I removed 5.5 from multi install and set single install to 5.6
- ran up stable system again

thanks!
